src/workspacer.Sound/workspacer.Sound.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<ItemGroup>
+		<ProjectReference Include="..\workspacer.Bar\workspacer.Bar.csproj" />
+		<ProjectReference Include="..\workspacer.Shared\workspacer.Shared.csproj" />
+	</ItemGroup>
+	<ItemGroup>
+	  <Reference Include="Vannatech.CoreAudio">
+	    <HintPath>Vannatech.CoreAudio.dll</HintPath>
```

Comment:
I'm not sure I like adding binaries to the project like this.

I would be much more comfortable with this if it was provided as a nuget-packages.

Can't you nudge the maintainers over at https://github.com/dorba/netcoreaudio to release this as a package?

Reply:
Definitely agree, would be a good addition

Reply:
That repo seems positively abondoned. No new activity in roughly 4 years. This fork seems more maintained though?

https://github.com/vuplea/CoreAudioAPI

Reply:
It's not very maintained, and doesnt have a nuget-package published.

How about using this library which seems to be popular (and maintained)?

https://www.nuget.org/packages/NAudio